docs/configure_https.md

```diff
@@ -120,3 +120,11 @@ After setting up HTTPS for Harbor, you can verify it by the follow steps:
     cp yourdomain.com.crt /etc/pki/ca-trust/source/anchors/reg.yourdomain.com.crt
     update-ca-trust
     ```
+
+3. If setting up harbor behind an nginx proxy or elastic load balancing, harbor nginx will reset the values of x-fowarded-XXX headers from first layer nginx or elastic load balancing, and the requests will not be routed properly.  
```

Comment:
a few comments:
1. Please sync with the latest version.
2. "harbor" should be "Harbor"
3. Is this for HTTPS, or for HTTP as well?
4. In which file to add in these lines?